pages/article/[id].js

```diff
@@ -146,97 +237,124 @@ function ArticlePage() {
+                  articleId={article.id}
+                  replyRequest={replyRequest}
+                  isArticleCreator={idx === 0}
+                />
+              ))}
+              <CreateReplyRequestForm
+                requestedForReply={article.requestedForReply}
+                articleId={article.id}
+                onNewReplyButtonClick={() => {
+                  setShowForm(true);
+                }}
+              />
+            </footer>
+          </Box>
+
+          {showForm && (
+            <div className={classes.newReplyContainer}>
+              <NewReplySection
+                article={article}
+                existingReplyIds={(article?.articleReplies || []).map(
+                  ({ replyId }) => replyId
+                )}
+                relatedArticles={article?.relatedArticles}
+                onSubmissionComplete={handleNewReplySubmit}
+                onClose={() => setShowForm(false)}
+              />
+            </div>
           )}
-          relatedArticles={article?.relatedArticles}
-          onSubmissionComplete={handleNewReplySubmit}
-        />
-      </section>
-
-      {article?.similarArticles?.edges?.length > 0 && (
-        <section className="section">
-          <h2>{t`You may be interested in the following similar messages`}</h2>
-          <ul className="similar-articles">
-            {article.similarArticles.edges.map(({ node }) => (
-              <ArticleItem key={node.id} article={node} />
-            ))}
-          </ul>
-        </section>
-      )}
-
-      <style jsx>{`
-        .section {
-          margin-bottom: 64px;
-        }
-        .header {
-          display: flex;
-          align-items: center;
-          flex-flow: row wrap;
-        }
-        .header > .trendline {
-          margin: 0 16px 0 auto;
-        }
-        .message {
-          border: 1px solid #ccc;
-          background: #eee;
-          border-radius: 3px;
-          padding: 24px;
-          word-break: break-all;
-        }
-        .items {
-          list-style-type: none;
-          padding-left: 0;
-        }
-        .similar-articles {
-          padding: 0;
-          list-style: none;
-        }
-      `}</style>
+
+          <Box
+            className={classes.card}
+            position="relative"
+            px={{ xs: 1.5, md: 3.5 }}
+            py={1.5}
+            mt={3}
+            id="current-replies"
+            ref={replySectionRef}
+          >
+            <h2>{t`${article.articleReplies.length} replies to the message`}</h2>
+            <Divider classes={{ root: classes.divider }} />
+            <CurrentReplies articleReplies={article.articleReplies} />
+          </Box>
+        </div>
+
+        <div className={cx(classes.card, classes.aside)}>
+          <h4>{t`Similar messages`}</h4>
+          {similarArticles.length ? (
+            <Box
+              display="flex"
+              flexDirection={{ xs: 'row', md: 'column' }}
+              overflow="scroll"
```

Comment:
Why overflow scroll here? It's introducing always-on horizontal & vertical scrolls.

What can we get from `overflow: scroll`?

![image](https://user-images.githubusercontent.com/108608/82547760-d33df080-9b8c-11ea-8849-65ffd3a13367.png)

Reply:
https://www.figma.com/file/zpD45j8nqDB2XfA6m2QskO/Cofacts-website?node-id=1307%3A285
the mobile version is horizontal scrolls

Reply:
I'll change it to `overflow: auto` though, to hide scrollbar when user using windows